packages/app/src/sandbox/eval/transpilers/sass/worker/sass-worker.js

```diff
@@ -28,6 +26,11 @@ async function compileSass(opts: ISassCompileOptions) {
 
   Sass._path = '/';
 
+  // TODO: Invalidate this in a smarter way
+  // we reset the found file cache and resolution cache in case one of the imports/filenames changed
+  const foundFileCache = {};
```

Comment:
Unfortunately we need to clear all the cache to have reliable builds when changing dependencies/imports

Gotta figure out more granular caching at some point... we do have full control due to the worker isolation and browser fs so we can probably go very far with this.